lib/backend/k8s/conversion/conversion.go

```diff
@@ -207,6 +190,10 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 		}
 	}
 
+	// Add the orchestrator label to the workload endpoint labels.
+	labels := pod.ObjectMeta.Labels
+	labels[apiv2.OrchestratorLabel] = apiv2.OrchestratorKubernetes
```

Comment:
I think your latest round erased my comment here, but I think it would be good to add the Namespace label to this map as well.

It means that people who query this through calicoctl will see it.  We probably want to do that in the CNI plugin as well so it works with etcd.